### PR TITLE
New Theme For Pico : Launcher

### PR DESCRIPTION
**Pull request by BesrourMS**

### CODE REVIEW

File: _themes/launcher.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+---
+heading: Launcher
+thumbnail: themes/images/thumbnails/launcher.jpg
+categories:
+  - single-page
+description: For those of us who aren't ready yet.
```
Comment (smcdougall) [inline _themes/launcher.md]: I'd like to change this so it's at least a little different than [Launch](http://picocms.org/themes/#entry-launch).

I'm thinking maybe "Launching in 3... 2... 1...".

These were originally meaningful descriptions, but at some point I just started writing some cheeky/fun text. :sweat_smile:

File: _themes/launcher.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
+---
+heading: Launcher
+thumbnail: themes/images/thumbnails/launcher.jpg
```
Comment (smcdougall) [inline _themes/launcher.md]: You haven't actually *included* these screenshots in your PR.  I can walk you through creating them if you'd like, or I can make them for you this one time. :wink:

I've been using Firefox's "Responsive Design Mode" to get the screenshot sizes right. (1440x900 for fullsize, 1080x880 for thumbnails, but then scaled down to 270x220).  I've also been saving them as 85% quality jpg for a little size optimization.

Up to you how you want to handle it.  I can provide the screenshots if necessary, but you submit Themes so much it'd be appreciated if you can provide your own. :smile:

**Comment by smcdougall** [comment]
Nice work.  As usual, thanks for your contribution.  :+1:

I've left a review for some things we need to fix, but it's nothing major.

Let me know if I can help. :smile:

File: _themes/launcher.md (hunk 2 above)
Comment (BesrourMS) [inline _themes/launcher.md]: Thanks @smcdougall, Please add the screenshots

Comment (smcdougall) [inline _themes/launcher.md]: Okay.  I won't be able to until this afternoon (morning here), so check back later.

I'll definitely make a point to get this added before tomorrow though. :+1:

**Pull request merged**